src/holochainClient.js

```diff
@@ -2,7 +2,12 @@ import { connect as hcWebClientConnect } from '@holochain/hc-web-client'
 import { get } from 'lodash/fp'
 import mockCallZome from 'mock-dnas/mockCallZome'
 
-export const MOCK_DNA_CONNECTION = true || process.env.NODE_ENV === 'test'
+export const MOCK_DNA_CONNECTION = false || process.env.NODE_ENV === 'test'
+export const MOCK_INDIVIDUAL_DNAS = {
```

Comment:
attn @JettTech @danekszy @lorenjohnson - Here's how I'm handling granular mocking of dnas. Note that if you set `MOCK_DNA_CONNECTION` above to true, this setting will be ignored.